test/dlc_tests/ec2/pytorch/training/test_pytorch_training.py

```diff
@@ -201,6 +204,9 @@ def test_pytorch_mpi_cpu(pytorch_training, ec2_connection, cpu_only, py3_only, e
     """
     Tests mpi backend
     """
+    _, framework_version = test_utils.get_framework_and_version_from_tag(pytorch_training)
+    if Version(framework_version) < Version("1.11.0"):
+        pytest.skip(f"MPI tests are skipped for PT<1.11 images")
```

Comment:
Please use fixture `pt111_and_above_only` instead of putting the skip as part of the test. By using the `pt111_and_above_only` to prevent the test function from executing, we can prevent test resources, such as EC2 instances, from being created.